adapter/src/main/java/org/jvalue/ods/adapterservice/datasource/model/Datasource.java

```diff
@@ -11,123 +12,121 @@
 
-    public DatasourceMetadata getMetadata() {
-      return metadata;
-    }
+  public DatasourceMetadata getMetadata() {
+    return metadata;
+  }
 
-    public DatasourceTrigger getTrigger() {
-      return trigger;
-    }
+  public DatasourceTrigger getTrigger() {
+    return trigger;
+  }
 
-    @Override
-    public String toString() {
-      return "Datasource{" +
-        "id=" + id +
-        ", protocol=" + protocol +
-        ", format=" + format +
-        ", metadata=" + metadata +
-        ", trigger=" + trigger +
-        '}';
-    }
+  @Override
+  public String toString() {
+    return "Datasource{" +
+      "id=" + id +
+      ", protocol=" + protocol +
+      ", format=" + format +
+      ", metadata=" + metadata +
+      ", trigger=" + trigger +
+      '}';
+  }
 
-    public AdapterConfig toAdapterConfig(RuntimeParameters runtimeParameters) {
-      DatasourceProtocol datasourceProtocol = fillQueryParameters(runtimeParameters);
-      return new AdapterConfig(
-        new ProtocolConfig(this.getProtocol().getType(), datasourceProtocol.getParameters()),
-        new FormatConfig(this.getFormat().getType(), this.getFormat().getParameters())
-      );
+  public AdapterConfig toAdapterConfig(RuntimeParameters runtimeParameters) {
+    var parameters = fillQueryParameters(runtimeParameters);
+    return new AdapterConfig(
+      new ProtocolConfig(this.getProtocol().getType(), parameters),
+      new FormatConfig(this.getFormat().getType(), this.getFormat().getParameters())
+    );
   }
 
-    protected DatasourceProtocol fillQueryParameters(RuntimeParameters runtimeParameters) {
-      if (!this.getProtocol().getType().equals("HTTP")) {
-        return this.getProtocol();
-      }
-      String url = (String) this.getProtocol().getParameters().get("location");
-      Map<String, String> defaultParameters = new HashMap<>();
-      if (this.getProtocol().getParameters().containsKey("defaultParameters")) {
-        defaultParameters = (Map<String, String>) this.getProtocol().getParameters().get("defaultParameters");
-      }
-      Map<String, String> triggerParameters = new HashMap<>();
-      if (runtimeParameters != null && runtimeParameters.parameters != null) {
-        triggerParameters = runtimeParameters.parameters;
-      }
-      Map<String, String> replacementParameters = Stream.of(defaultParameters, triggerParameters).flatMap(map -> map.entrySet().stream()).collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue, (v1, v2) -> v2));
-      for (Map.Entry<String, String> parameter : replacementParameters.entrySet()) {
-        url = url.replace("{" + parameter.getKey() + "}", parameter.getValue());
-      }
-      HashMap<String, Object> parameters = new HashMap<>(this.getProtocol().getParameters());
-      parameters.put("location", url);
-      return new DatasourceProtocol(this.getProtocol().getType(), parameters);
+  protected Map<String, Object> fillQueryParameters(RuntimeParameters runtimeParameters) {
```

Comment:
This method looks very crowded.
Could you please add a blank line between the different `Map<>...` statements.

Reply:
Well, this method was there before, I have just adjusted the return type and changed the indentation from 4 to 2 spaces. But I will try to make this method more beautiful.

Reply:
Hi, I know you didn't created the method. But when you touch it you can also leave it cleaner than before.
The Boy Scout Rule 🙃

Reply:
I have already refactored it and it is much cleaner now

Reply:
Looks better now. Thanks